Lib/http/server.py

```diff
@@ -1052,7 +1072,8 @@ def server_bind(self):
 
         def finish_request(self, request, client_address):
             self.RequestHandlerClass(request, client_address, self,
-                                     directory=args.directory)
+                                     directory=args.directory,
+                                     response_headers=response_headers)
```

Comment:
```suggestion
                                     response_headers=args.header)
```

Reply:
Updated as above in d317cc2d333f7b8b340e5133378cdc64b5b377e7